docgen/src/guides/v3-migration.md

```diff
@@ -434,9 +434,9 @@ With the redo button:
           <svg width="16" height="16" viewBox="0 0 38 38" xmlns="http://www.w3.org/2000/svg" stroke="#444" class="ais-SearchBox-loadingIcon">
             <g fill="none" fillRule="evenodd">
               <g transform="translate(1 1)" strokeWidth="2">
-                <circle strokeOpacity=".5" cx="18" cy="18" r="18" />
+                <circle stroke-opacity=".5" cx="18" cy="18" r="18" />
                 <path d="M36 18c0-9.94-8.06-18-18-18">
-                  <animateTransform
+                  <animate-transform
                     attributeName="transform"
```

Comment:
this too, fillRule, viewBox, strokeWidth :)

Reply:
They are valid attributes.